docs/src/config_updates.md

```diff
@@ -24,3 +24,13 @@ is set to `false`.
     auto_update = true
     auto_update_interval_hours = 24
 
+### archive_url
+
+URL for the location of tldr pages archive. Default is the main `thdr.sh`
+archive location.
```

Comment:
The new client specification has changed the recommended source for pages again, instead of using `tldr.sh`, the Github releases are preferred now. I would thus change this to
```suggestion
URL for the location of tldr pages archive. By default the pages are fetched from the latest `tldr-pages/tldr` GitHub release.
```